spec/state_machines/foi_permitted_events_spec.rb

```diff
@@ -0,0 +1,193 @@
+require 'rails_helper'
+
+describe 'FOI permittted events' do
```

Comment:
This should specify what method is being tested: `ConfigurableStateMachine::permitted_events` (I think?)